python/lsst/ts/standardscripts/scheduler/standby.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of ts_standardscripts.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["Standby"]
+
+from lsst.ts import salobj
+from lsst.ts.idl.enums.Scheduler import SalIndex
+
+from .set_desired_state import SetDesiredState
+
+
+class Standby(SetDesiredState):
```

Comment:
I suggest ditching this and just subclassing SetDesiredState directly in the new scripts. It's just one more constructor argument and saves two classes that don't seem to be very useful.

Reply:
I removed the Standby one but I left the Enabled one. Mostly because the later has configuration in it, which means I would duplicate the code on both auxtel/maintel version and would not be able to unit test it properly.